src/Raven.Server/Documents/Subscriptions/Processor/DocumentsDatabaseSubscriptionProcessor.cs

```diff
@@ -39,6 +39,10 @@ public override async Task<SubscriptionBatchResult> GetBatchAsync(SubscriptionBa
 
             foreach (var item in Fetcher.GetEnumerator())
             {
+                // if it's a tryout, we need to skip archived/unarchived documents at this level  
+                if (SubscriptionConnectionsState is DummySubscriptionConnectionsState connectionsState && connectionsState.ShouldSkipDocumentFromTryout(item.Flags)) 
+                    continue;
```

Comment:
can we move this to `TryoutSubscriptionAsync` ?

Reply:
replaced with a different solution
now we simply don't send documents that were filtered by [`ShouldSend` method ](https://github.com/ravendb/ravendb/blob/d3ffe231bebabedbf0376c263b4ea568cc0957c1/src/Raven.Server/Documents/Subscriptions/Processor/DocumentsDatabaseSubscriptionProcessor.cs#L126)